### DISCUSSION TITLE
Backend access securization

### DISCUSSION THREAD

**Discussion by shogd3838** [AzuraCast/AzuraCast#4475] [Q&A]
Hello everyone.
I ask the community because I think I'm not the first one having this need and I didn't find anything related in the official documentation.

I have a radio station using the docker installation of azuracast and I want to limit the IP addresses that can access the backend web pages. I think limiting IP only for the page /login is enough. 
As it is a docker installation I don't feel very confident in modifying anything.
Also if it's possible I'm also searching how to redirect / to /public/<page of my radio> in order to access the webpage of the radio only by typing the domain URL.

I will be very grateful if anyone can give me some advices on those topics.

Thanks.

**Comment by SmartGuyCodes** [comment]
I suggest use .htaccess to limit access to specific IP addresses to the admin login

**Comment by joshopkins787** [comment]
You would have to make a custom nginx configuration for the docker. That had a location section for the login.

**Reply by shogd3838** [reply]
Hi thanks. That's why I think could do the job but my knowledge in nginx especially in docker is quite null. I'm more used to apache or tomcat conf.
Do you think it's possible to do this via the docker-override file of azuracast ?

**Reply by joshopkins787** [reply]
It should be. I was reading the docs and there is a section about make changes to the configs within the docker.
